### DISCUSSION TITLE
E3V2-427 Board With BL touch already Installed

### DISCUSSION THREAD

**Discussion by nuttster2** [Jyers/Marlin#825] [Q&A] [answered]
When I load one of the Bin Files for my board 427 with the 5 x 5 bl touch , It loads all the files according to the screen functions
I then go to do a function and it does nothing, in fact it does not function at all , it like it is locked up.
You can hear the motors engage but nothing moves
I do have the stock Creality firmware V1.1.1 for the BL touch on my printers.
What am i doing wrong ??
Please help I would really love have this Firmware on both my printers.
NewBee

**Comment by smooreace** [comment, marked as answer]
Are you sure you have a 4.2.7 board?

**Reply by nuttster2** [reply]
I will pull it apart tonight when i get home from work a double check
and get back to you tomorrow Saturday
Thank you for your reply.....

**Reply by nuttster2** [reply]
I went a head and started from scratch and reloaded The factory  stock bin. file for the 4.2.7 board from Creality 
And now its working fine, I don't know what happened the first few times but now it working Great !!
Love the new firmware very helpful for new people 
Thank you
